### PR TITLE
DisVoice docset

### PR DESCRIPTION
**Pull request by NeuroForLunch**

### CODE REVIEW

File: docsets/DisVoice/docset.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "name": "DisVoice",
+    "version": "0.1.1",
+    "archive": "DisVoice.tgz",
```
Comment (Kapeli) [inline docsets/DisVoice/docset.json:4]: This archive does not exist. You added `disvoice.tgz` instead

**Comment by Kapeli** [comment]
`icon.png` needs to be `16x16` in size and `icon@2x.png` needs to be `32x32`

**Comment by NeuroForLunch** [comment]
fixed

**Comment by Kapeli** [comment]
Thank you! 👍

**Pull request merged**